modules/hrm/includes/class-ajax.php

```diff
@@ -1429,13 +1438,13 @@ public function employee_education_create() {
         }
 
         $edu_id   = isset( $_POST['edu_id'] ) ? intval( $_POST['edu_id'] ) : 0;
-        $school   = isset( $_POST['school'] ) ? strip_tags( sanitize_text_field( wp_unslash( $_POST['school'] ) ) ) : '';
-        $degree   = isset( $_POST['degree'] ) ? strip_tags( sanitize_text_field( wp_unslash( $_POST['degree'] ) ) ) : '';
-        $field    = isset( $_POST['field'] ) ? strip_tags( sanitize_text_field( wp_unslash( $_POST['field'] ) ) ) : '';
+        $school   = isset( $_POST['school'] ) ? wp_strip_all_tags( sanitize_text_field( wp_unslash( $_POST['school'] ) ) ) : '';
```

Comment:
kindly remove all the occurrences of wp_strip_all_tags where sanitize_text_field() function is used, becase sanitize_text_field() function does that already: https://developer.wordpress.org/reference/functions/_sanitize_text_fields/